test/infra/infra-mysql56-single/docker-compose-init.bash

```
@@ -0,0 +1,108 @@
+#!/bin/bash
+if [ -z "${INFRA_ID}" ]; then
+    export INFRA_ID=$(basename $(dirname $(pwd)) | sed 's/infra-//; s/docker-//')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)